float64 time
float64 enc_left
float64 enc_right
